types/hero.ts

```diff
@@ -113,15 +113,40 @@ export interface Equipment {
   name: string;
   slot: EquipmentSlot;
   type: EquipmentType;
+  effect: EquipmentEffect;
+  stat1: EquipmentStat | null;
+  stat2: EquipmentStat | null;
+  notes: string;
+}
+
+export interface ExclusiveEquipment {
+  name: string;
+  slot: string;
   effect: string;
-  stat1: string | null;
-  stat2: string | null;
+}
```

Comment:
Same deal here, tbh though. In an ideal world an ExclusiveEquipment would have all the properties of an Equipment with an additional `hero` field to denote what hero it is used by.

Reply:
I absolutely agree ExclusiveEquipment  should be part of Equipment. But its db design problem. We just had to add all exclusive items into Equipment worksheet and only make link  "item-hero" on hero worksheet.
That would let us make exclusive equipment  list on Equipment page. But after we added  stat/effect levels into Equipment  its become really different entities